src/Sylius/Behat/Context/Api/Admin/ManagingProductsContext.php

```diff
@@ -424,4 +519,30 @@ private function hasProductImage(Response $response, ProductInterface $product):
 
         return false;
     }
+
+    private function getFieldValueOfFirstProduct(array $product, string $field): ?string
```

Comment:
This function contains technical details of the response - we should definitely use some functions from the `ResponseChecker` and/or adjust them/create new ones